src/main/java/app/JeM.java

```diff
@@ -0,0 +1,169 @@
+package app;
+
+import taskmanager.Storage;
+import exceptions.InvalidCommandException;
+import tasks.Task;
+import tasks.Todo;
+import tasks.Deadline;
+import tasks.Event;
+
+import java.util.Scanner;
+
+public class JeM {
+
+    public static void main(String[] args) {
+        displayWelcomeMessage();
+
+        Scanner scanner = new Scanner(System.in);
+        Storage storage = new Storage();
+
+        while (true) {
+            String line = scanner.nextLine().trim();
+            if (line.equalsIgnoreCase("bye")) {
+                exitChatBot();
+                break;
+            }
+            try {
+                handleInput(line, storage);
+            }catch(InvalidCommandException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+        printBreakLine();
+        scanner.close();
+    }
+
+    private static void displayWelcomeMessage() {
+        String logo = "      _         __  __ \n"
+                + "     | |       |  \\/  |\n"
+                + "     | |  ___  | |\\/| |\n"
+                + " _   | | / _ \\ | |  | |\n"
+                + "| |__| ||  __/ | |  | |\n"
+                + " \\____/  \\___| |_|  |_|\n";
+        System.out.println("Hello from\n" + logo);
+
+        printBreakLine();
+
+        System.out.println(" Hello! I'm JeM, Your e-Assistant");
+        System.out.println(" Personal To-Do list bot! ");
+        System.out.println(" Just type out your tasks you have to complete and I will make a list of them for you.");
+        System.out.println(" For tasks with no deadline, type todo <task name>, for tasks with deadlines, type deadline <task name> /by <date and time>");
+        System.out.println(" For events, type event <task name> /from <date and time> /to <date and time>");
+        System.out.println(" Type 'list' to see the current list of tasks,'delete <task number>' to delete that task");
+        System.out.println(" Finally, type 'bye' to end the chat!");
+
+        printBreakLine();
+    }
+
+    private static void handleInput(String line, Storage storage) throws InvalidCommandException {
+        if (line.equalsIgnoreCase("list")) {
+            storage.storageList();
+        } else if (line.toLowerCase().startsWith("delete")) {
+            handleDeleteCommand(line, storage);
+        } else if (line.toLowerCase().startsWith("unmark")) {
+            handleUnmarkCommand(line, storage);
+        } else if (line.toLowerCase().startsWith("mark")) {
+            handleMarkCommand(line, storage);
+        } else if (line.toLowerCase().startsWith("todo")) {
+            handleTodoCommand(line, storage);
+        } else if (line.toLowerCase().startsWith("deadline")) {
+            handleDeadlineCommand(line, storage);
+        } else if (line.toLowerCase().startsWith("event")) {
+            handleEventCommand(line, storage);
```

Comment:
.toLowerCase() seems to be used repetitively here. A switch case may be used instead.